R/startModel.R

```diff
@@ -9,64 +9,29 @@
 #'
 #' @param config run configurations
 #' @param path character vector with folders to run the model in
-#' @param outputFolder directory of output folder
-#' @param references named list of matching references
 #' @export
 #'
-startModel <- function(config = NULL,
-                       path = NULL,
-                       outputFolder = NULL,
-                       references = NULL) {
+startModel <- function(config,
+                       path) {
 
-  # if no outputFolder is defined, use this or create it in current wd
-  defaultOutputFolderName <- "output"
+  cfg <- readConfig(config, readDirect = TRUE)
```

Comment:
Same question regarding `readDirect` here. Also, I think we should only use the given config if this is not a restart and read the already existing one else (and warn that the given config is ignored).

Reply:
As above: readDirect is kept for the moment. In startModel we always read the config from path/config/config.yaml; this was already the case before (implemented in startScriptSlurm), but I now also removed config as an input to startModel to avoid confusion.